.travis.yml

```diff
@@ -13,12 +13,19 @@ python:
 stages:
   - lint
   - test
+  - name: deploy
+    if: tag is present
```

Comment:
If I understand correctly, this is done so a new tag will cause a deployment, right?

Reply:
Hello @carlosalberto,
Right. Any new tag pushed to GitHub will cause a deployment.